client/my-sites/checkout/checkout-thank-you/jetpack-plan-details.jsx

```diff
@@ -4,85 +4,133 @@
+		const { selectedSite, trackManualInstall, translate } = this.props;
 
-		props.title = null;
 		if ( ! selectedSite.canUpdateFiles ) {
-			props.description = i18n.translate(
-				'You can now install Akismet and VaultPress, which will automatically protect your site from spam and data loss. ' +
-					"If you have any questions along the way, we're here to help!"
-			);
-			props.buttonText = i18n.translate( 'Installation Instructions' );
-			props.href = 'https://en.support.wordpress.com/setting-up-premium-services/';
-			props.onClick = trackManualInstall( 'calypso_plans_autoconfig_click_manual_install' );
-		} else {
-			props.description = i18n.translate(
-				'We are about to install Akismet and VaultPress for your site, which will automatically protect your site from spam ' +
-					"and data loss. If you have any questions along the way, we're here to help! You can also perform a manual " +
-					'installation by following {{a}}these instructions{{/a}}.',
-				{
-					components: {
-						a: (
-							<a
-								target="_blank"
-								rel="noopener noreferrer"
-								href="https://en.support.wordpress.com/setting-up-premium-services/"
-								onClick={ trackManualInstall( 'calypso_plans_autoconfig_click_opt_out' ) }
-							/>
-						),
-					},
-				}
+			return (
+				<PurchaseDetail
+					icon="cog"
+					buttonText={ translate( 'Installation Instructions' ) }
+					description={ translate(
+						'You can now install Akismet and VaultPress, which will automatically ' +
+							'protect your site from spam and data loss. ' +
+							"If you have any questions along the way, we're here to help!"
+					) }
+					href="https://en.support.wordpress.com/setting-up-premium-services/"
+					onClick={ trackManualInstall }
+				/>
 			);
-			props.buttonText = i18n.translate( 'Set up your plan' );
-			props.href = `/plugins/setup/${ selectedSite.slug }`;
 		}
+
+		return (
+			<PurchaseDetail
+				icon="cog"
+				buttonText={ translate( 'Set up your plan' ) }
+				description={ translate(
+					'We are about to install Akismet and VaultPress for your site, ' +
+						'which will automatically protect your site from spam ' +
+						'and data loss. If you have any questions along the way, ' +
+						"we're here to help! You can also perform a manual " +
+						'installation by following {{a}}these instructions{{/a}}.',
+					{
+						components: {
+							a: (
+								<a
+									target="_blank"
+									rel="noopener noreferrer"
+									href="https://en.support.wordpress.com/setting-up-premium-services/"
+									onClick={ trackManualInstall }
+								/>
+							),
+						},
+					}
+				) }
+				href={ `/plugins/setup/${ selectedSite.slug }` }
+			/>
+		);
+	}
+}
+
+const getTracksDataForAutoconfigHalt = selectedSite => {
+	const reasons = utils.getSiteFileModDisableReason( selectedSite, 'modifyFiles' );
+
+	if ( reasons && reasons.length > 0 ) {
+		return [ 'calypso_plans_autoconfig_halt_filemod', { error: reasons[ 0 ] } ];
```

Comment:
As mentioned in one of the review comments, this one could be more readable if it's an object:

```
return {
	name: 'calypso_plans_autoconfig_halt_filemod',
	properties: { error: reasons[ 0 ] },
};
```

Reply:
reconfigured in 2ec88af712